bsg_dmc/bsg_dmc.vh

```diff
@@ -1,9 +1,15 @@
-
 `ifndef BSG_DMC_VH
 `define BSG_DMC_VH
 
   `include "bsg_defines.v"
 
+  `define declare_dmc_trace_entry_s(addr_width_mp, burst_width_mp, data_width_mp) \
+  	typedef struct packed { \
+  		app_cmd_e cmd; \
+  		logic [addr_width_mp - 1:0] addr; \
+  		logic [burst_width_mp*data_width_mp -1 :0] data; \
```

Comment:
The trace should have the burst data in serial.
[ cmd addr ]
[ data ]
[ data ]
[ data ]

Otherwise, because we're tunnelling the connection, it will cause commands without payloads to take 8x more bandwidth as we ser/des a burst length packet